
The new battery driver was written with Series 5 meassurements in mind - so
here I add some guess first-shoot values for the Revo familly.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.4.27/drivers/ssi/ads7843.c.vanilla	2005-02-07 10:38:46.864858176 +0100
+++ linux-2.4.27/drivers/ssi/ads7843.c	2005-02-07 10:45:28.858745816 +0100
@@ -98,10 +98,17 @@
 
 
 //Assume default NiMh battery
+#if defined(CONFIG_PSIONW_REVO) || defined(CONFIG_PSIONW_REVOPLUS)
+static int bat_max_voltage = 3600; //[mV]
+static int bat_min_voltage = 2800; //[mV]
+static int bat_capacity = 800; //[mAh]
+static int bat_avg_current = 20; //[mA]
+#else
 static int bat_max_voltage = 2600; //[mV]
 static int bat_min_voltage = 2400; //[mV]
 static int bat_capacity = 1300; //[mAh]
 static int bat_avg_current = 100; //[mA]
+#endif
 
 #define BAT_OFF 1800 //1.8V psion is switched off
 
